tests/unit/obmTest.py

```diff
@@ -0,0 +1,87 @@
+"""Unit tests for api.py"""
```

Comment:
This test file has absolutely nothing to do with the content of this patch; it doesn't exercise the CLI at all. Explain?